radicle-cli/src/commands/assign.rs

```diff
@@ -0,0 +1,99 @@
+use std::ffi::OsString;
+use std::str::FromStr;
+
+use anyhow::anyhow;
+
+use radicle::cob::issue;
+use radicle::identity;
+
+use crate::terminal as term;
+use crate::terminal::args::{Args, Error, Help};
+use radicle::cob;
+use radicle::cob::issue::Issues;
+use radicle::storage::WriteStorage;
+
+pub const HELP: Help = Help {
+    name: "assign",
+    description: "Manage issues",
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+
+    rad assign <issue> @<alias>
```

Comment:
What's the `<alias>` for? And wouldn't you also need to parse the `ActorId`? :thinking:

Reply:
The alias is shorthand for another peer (radicle user).  Support will be added in upcoming changes, its been left out here.  For now, its the ActorID you mentioned.